### PR TITLE
[hi] Translate `Cloud Native Security` into `Hindi`

### PR DESCRIPTION
**Pull request by imvisp**
- Updated the cloud native security term.
- Ref #686

### CODE REVIEW

File: content/hi/cloud_native_security.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Category: अवधारणा 
+
+--- 
+
+ 
+
+## यह क्या है 
+
+क्लाउड नेटिव सुरक्षा एक ऐसा दृष्टिकोण है जो सुरक्षा को [क्लाउड नेटिव एप्लिकेशन] (/cloud_native_apps/) में बनाता है। यह सुनिश्चित करता है कि सुरक्षा विकास के संपूर्ण एप्लिकेशन जीवनचक्र का हिस्सा है। क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा मॉडल के समान मानकों को सुनिश्चित करने का प्रयास करती है, जबकि क्लाउड नेटिव वातावरण के विवरणों को अपनाती है, अर्थात् तेजी से कोड परिवर्तन और अत्यधिक अल्पकालिक बुनियादी ढाँचा। क्लाउड नेटिव सुरक्षा `DevSecOps` नामक अभ्यास से अत्यधिक संबंधित है।
```
Comment (jihoon-seo) [inline content/hi/cloud_native_security.md:15]: To make the link work:

```suggestion
क्लाउड नेटिव सुरक्षा एक ऐसा दृष्टिकोण है जो सुरक्षा को [क्लाउड नेटिव एप्लिकेशन](/cloud_native_apps/) में बनाता है। यह सुनिश्चित करता है कि सुरक्षा विकास के संपूर्ण एप्लिकेशन जीवनचक्र का हिस्सा है। क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा मॉडल के समान मानकों को सुनिश्चित करने का प्रयास करती है, जबकि क्लाउड नेटिव वातावरण के विवरणों को अपनाती है, अर्थात् तेजी से कोड परिवर्तन और अत्यधिक अल्पकालिक बुनियादी ढाँचा। क्लाउड नेटिव सुरक्षा `DevSecOps` नामक अभ्यास से अत्यधिक संबंधित है। 

```

**Review by jihoon-seo — COMMENTED** [review]
Preview: https://deploy-preview-687--cncfglossary.netlify.app/hi/cloud_native_security/

@imvisp Welcome, and thank you for your first contribution!
And I left an comment. PTAL!

**Comment by imvisp** [comment]
> 

Made changes as per you suggested :+1: 
Thank you :)

**Review by jayesh-srivastava — APPROVED** [review]
lgtm

File: content/hi/cloud_native_security.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## समस्या 
+
+पारंपरिक सुरक्षा मॉडल कई मान्यताओं के साथ बनाए गए थे जो अब मान्य नहीं हैं। क्लाउड नेटिव ऐप बार-बार बदलते हैं, बड़ी संख्या में ओपन सोर्स टूल और लाइब्रेरी का उपयोग करते हैं, जो अक्सर विक्रेता-नियंत्रित बुनियादी ढांचे में चलते हैं, और तेजी से बुनियादी ढांचे में बदलाव के अधीन हैं। कोड समीक्षाएं, लंबे गुणवत्ता आश्वासन चक्र, होस्ट-आधारित भेद्यता स्कैनिंग, और अंतिम मिनट की सुरक्षा समीक्षाएं क्लाउड नेटिव एप्लिकेशन के साथ स्केल नहीं हो सकती हैं। 
+
+ 
+
+## समाधान 
+
+क्लाउड नेटिव सुरक्षा काम करने का एक नया तरीका पेश करती है जो पारंपरिक सुरक्षा मॉडल से एक में माइग्रेट करके अनुप्रयोगों की सुरक्षा करती है जहां सुरक्षा रिलीज चक्र के हर चरण में शामिल होती है। मैनुअल ऑडिट और चेक को बड़े पैमाने पर स्वचालित स्कैन से बदल दिया जाता है। रैपिड कोड रिलीज़ पाइपलाइनों को ऐसे टूल के साथ एकीकृत किया जाता है जो संकलित होने से पहले कमजोरियों के लिए कोड स्कैन करते हैं। ओपन सोर्स लाइब्रेरी को विश्वसनीय स्रोतों से खींचा जाता है और कमजोरियों के लिए निगरानी की जाती है। परिवर्तन को धीमा करने के बजाय एक क्लाउड नेटिव सुरक्षा मॉडल इसे बार-बार अपडेट किए जाने वाले कमजोर घटकों द्वारा गले लगाता है या यह सुनिश्चित करता है कि बुनियादी ढांचे को नियमित रूप से बदल दिया जाए।
```
Comment (Garima-Negi) [inline content/hi/cloud_native_security.md:27]:
```suggestion
क्लाउड नेटिव सुरक्षा काम करने का एक नया तरीका पेश करती है जो पारंपरिक सुरक्षा मॉडल से एक में माइग्रेट करके अनुप्रयोगों की सुरक्षा करती है जहां सुरक्षा रिलीज चक्र के हर चरण में शामिल होती है। मैनुअल ऑडिट और चेक को बड़े पैमाने पर स्वचालित स्कैन से बदल दिया जाता है। रैपिड कोड रिलीज़ पाइपलाइनों को ऐसे टूल के साथ एकीकृत किया जाता है जो संकलित होने से पहले कमजोरियों के लिए कोड स्कैन करते हैं। ओपन सोर्स लाइब्रेरी को विश्वसनीय स्रोतों से खींचा जाता है और कमजोरियों के लिए निगरानी की जाती है। परिवर्तन को धीमा करने के बजाय एक क्लाउड नेटिव सुरक्षा मॉडल इसे बार-बार अपडेट किए जाने वाले कमजोर घटकों द्वारा शामिल करता है या यह सुनिश्चित करता है कि बुनियादी ढांचे को नियमित रूप से बदल दिया जाए। 

```

**Review by Garima-Negi — CHANGES_REQUESTED** [review]

File: content/hi/cloud_native_security.md (hunk 2 above)
Comment (imvisp) [reply content/hi/cloud_native_security.md:27]: Okay, will do that 👍

Thank you ☺️

**Comment by imvisp** [comment]
I have made changes as per you suggested 

Thank you @Garima-Negi

File: content/hi/cloud_native_security.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+--- 
+
+Title: क्लाउड नेटिव सुरक्षा 
+
+Status: completed 
+
+Category: अवधारणा 
+
+---
```
Comment (anubha-v-ardhan) [inline content/hi/cloud_native_security.md:9]:
```suggestion
--- 
Title: क्लाउड नेटिव सुरक्षा (Cloud Native Security)
Status: completed 
Category: अवधारणा 
--- 

```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

File: content/hi/cloud_native_security.md (hunk 3 above)
Comment (imvisp) [reply content/hi/cloud_native_security.md]: Made changes as per you suggested

Thank you @anubha-v-ardhan :+1:

File: content/hi/cloud_native_security.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+Status: completed 
+Category: अवधारणा 
+
+--- 
+
+ 
+
+## यह क्या है 
+
+क्लाउड नेटिव सुरक्षा एक ऐसा दृष्टिकोण है जो सुरक्षा को [क्लाउड नेटिव एप्लिकेशन](/cloud_native_apps/) में बनाता है। यह सुनिश्चित करता है कि सुरक्षा विकास के संपूर्ण एप्लिकेशन जीवनचक्र का हिस्सा है। क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा मॉडल के समान मानकों को सुनिश्चित करने का प्रयास करती है, जबकि क्लाउड नेटिव वातावरण के विवरणों को अपनाती है, अर्थात् तेजी से कोड परिवर्तन और अत्यधिक अल्पकालिक बुनियादी ढाँचा। क्लाउड नेटिव सुरक्षा `DevSecOps` नामक अभ्यास से अत्यधिक संबंधित है।
```
Comment (Garima-Negi) [inline content/hi/cloud_native_security.md:13]:
```suggestion
 क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा मॉडल के समान मानकों को सुनिश्चित करने के साथ-साथ क्लाउड नेटिव वातावरण के विवरणों के अनुकूल होने का प्रयास करती है, अर्थात् तेजी से कोड परिवर्तन और अत्यधिक अल्पकालिक बुनियादी ढाँचा। 
```
translation for while is `जबकि` but it doesn't fit the meaning. I've added a suggestion here. Please take a look and incorporate if it makes sense.

File: content/hi/cloud_native_security.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+## समस्या 
+
+पारंपरिक सुरक्षा मॉडल कई मान्यताओं के साथ बनाए गए थे जो अब मान्य नहीं हैं। क्लाउड नेटिव ऐप बार-बार बदलते हैं, बड़ी संख्या में ओपन सोर्स टूल और लाइब्रेरी का उपयोग करते हैं, जो अक्सर विक्रेता-नियंत्रित बुनियादी ढांचे में चलते हैं, और तेजी से बुनियादी ढांचे में बदलाव के अधीन हैं। कोड समीक्षाएं, लंबे गुणवत्ता आश्वासन चक्र, होस्ट-आधारित भेद्यता स्कैनिंग, और अंतिम मिनट की सुरक्षा समीक्षाएं क्लाउड नेटिव एप्लिकेशन के साथ स्केल नहीं हो सकती हैं। 
+
+ 
+
+## समाधान 
+
+क्लाउड नेटिव सुरक्षा काम करने का एक नया तरीका पेश करती है जो पारंपरिक सुरक्षा मॉडल से एक में माइग्रेट करके अनुप्रयोगों की सुरक्षा करती है जहां सुरक्षा रिलीज चक्र के हर चरण में शामिल होती है। मैनुअल ऑडिट और चेक को बड़े पैमाने पर स्वचालित स्कैन से बदल दिया जाता है। रैपिड कोड रिलीज़ पाइपलाइनों को ऐसे टूल के साथ एकीकृत किया जाता है जो संकलित होने से पहले कमजोरियों के लिए कोड स्कैन करते हैं। ओपन सोर्स लाइब्रेरी को विश्वसनीय स्रोतों से खींचा जाता है और कमजोरियों के लिए निगरानी की जाती है। परिवर्तन को धीमा करने के बजाय एक क्लाउड नेटिव सुरक्षा मॉडल इसे बार-बार अपडेट किए जाने वाले कमजोर घटकों द्वारा शामिल करता है या यह सुनिश्चित करता है कि बुनियादी ढांचे को नियमित रूप से बदल दिया जाए।
```
Comment (Garima-Negi) [inline content/hi/cloud_native_security.md:25]: The first sentence seems a little misplaced. Here's a suggestion to write it in a different way :
```क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा मॉडल को एक ऐसे मॉडल में माइग्रेट करती है जहां सुरक्षा रिलीज़ चक्र के हर चरण में शामिल होती है, इस प्रकार एप्लिकेशन की सुरक्षा का एक नया तरीका पेश करती है।```

**Review by Garima-Negi — CHANGES_REQUESTED** [review]

File: content/hi/cloud_native_security.md (hunk 4 above)
Comment (imvisp) [reply content/hi/cloud_native_security.md:13]: Okay, make sense!!
 this space is necessary right ? at the starting of the sentence
```
 क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा मॉडल के समान मानकों को सुनिश्चित करने के साथ-साथ क्लाउड नेटिव वातावरण के विवरणों के अनुकूल होने का प्रयास करती है, अर्थात् तेजी से कोड परिवर्तन और अत्यधिक अल्पकालिक बुनियादी ढाँचा। 

```

File: content/hi/cloud_native_security.md (hunk 5 above)
Comment (imvisp) [reply content/hi/cloud_native_security.md:25]: Fixed, let me know any changes!!

```suggestion
क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा मॉडल को एक ऐसे मॉडल में माइग्रेट करती है जहां सुरक्षा रिलीज़ चक्र के हर चरण में शामिल होती है, इस प्रकार एप्लिकेशन की सुरक्षा का एक नया तरीका पेश करती है। मैनुअल ऑडिट और चेक को बड़े पैमाने पर स्वचालित स्कैन से बदल दिया जाता है। रैपिड कोड रिलीज़ पाइपलाइनों को ऐसे टूल के साथ एकीकृत किया जाता है जो संकलित होने से पहले कमजोरियों के लिए कोड स्कैन करते हैं। ओपन सोर्स लाइब्रेरी को विश्वसनीय स्रोतों से खींचा जाता है और कमजोरियों के लिए निगरानी की जाती है। परिवर्तन को धीमा करने के बजाय एक क्लाउड नेटिव सुरक्षा मॉडल इसे बार-बार अपडेट किए जाने वाले कमजोर घटकों द्वारा शामिल करता है या यह सुनिश्चित करता है कि बुनियादी ढांचे को नियमित रूप से बदल दिया जाए। 

```

File: content/hi/cloud_native_security.md (hunk 4 above)
Comment (Garima-Negi) [reply content/hi/cloud_native_security.md:13]: you can get rid of the whitespace!  must have been a copy/paste addition

File: content/hi/cloud_native_security.md (hunk 5 above)
Comment (Garima-Negi) [reply content/hi/cloud_native_security.md]: looks good

**Review by Garima-Negi — APPROVED** [review]

**Comment by jayesh-srivastava** [comment]
@imvisp 5 commits would be unnecessary. Please consider squashing the commits into one.

**Comment by imvisp** [comment]
> @imvisp 5 commits would be unnecessary. Please consider squashing the commits into one.

Can you pls tell me how can i do that ?

**Comment by jayesh-srivastava** [comment]
@imvisp This will give you a clear idea. 
https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/managing-commits/squashing-commits

**Comment by imvisp** [comment]
> @imvisp This will give you a clear idea. https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/managing-commits/squashing-commits

In the selection of branch i have to selecet the `dev-hi` branch right ?

**Comment by jayesh-srivastava** [comment]
@imvisp Seems like you have unmerged commits coming from your branch which are 250 in number. I would recommend making a new branch for every PR that you make and avoid using your main branch. Also requesting review from other localization's maintainers is unnecessary. I am closing this PR for now. Try to fix issues in this else feel free to open a new PR.

**Comment by imvisp** [comment]
> @imvisp Seems like you have unmerged commits coming from your branch which are 250 in number. I would recommend making a new branch for every PR that you make and avoid using your main branch. Also requesting review from other localization's maintainers is unnecessary. I am closing this PR for now. Try to fix issues in this else feel free to open a new PR.

I'm new to contribution that's why I'm not able to understand what you're trying to say, i was checking out some way and i didn't even know how i requested all the maintainers.

**Comment by imvisp** [comment]
> @imvisp Seems like you have unmerged commits coming from your branch which are 250 in number. I would recommend making a new branch for every PR that you make and avoid using your main branch. Also requesting review from other localization's maintainers is unnecessary. I am closing this PR for now. Try to fix issues in this else feel free to open a new PR.

Can you please tell me how can do that ?

**Comment by imvisp** [comment]
Can you pls tell me that what should I do and how can I contribute in this file ?

**Pull request closed without merging**